wharehauora-wifi-gateway/wharehauora-wifi-gateway.ino

```diff
@@ -2,25 +2,45 @@
 #include <SPI.h>
 #include <WiFiManager.h>         //https://github.com/tzapu/WiFiManager
 #include <ESP8266WiFi.h>
+#include <Ticker.h>
+
+#define FIRMWAREVERSION "VERSION GOES HERE"
```

Comment:
Wanna put a version in there? We'll increment it as we merge

Reply:
Yeah totally, I was looking for an automatic way to do this upon git commit - but its kind of a chicken and egg problem and its kinda tricky to do this server-side on the git server.

I'll stop looking for an automatic way, and we can just do it manually.